Zapper-fi/studio
# feat(rubicon): Add Rubicon.finance and Rubicon Bath Tokens

## Description

This is an initial PR to initialize [Rubicon](https://www.rubicon.finance/) as a Zapper project. We are excited to integrate with Zapper and this PR represents a basic implementation that tracks [Bath Tokens](https://docs.rubicon.finance/rubicon-docs/contracts/rubicon-pools/bath-token) and their underlying value; the same way we do in our [app](https://app.rubicon.finance/).

## Checklist

- [X] (optional) As a contributor, my Ethereum address/ENS is: `bghughes.eth`
- [X] (optional) As a contributor, my Twitter handle is: `@_benjaminhughes`

## How to test?

A sanity check on the ERC-20 Bath Token balances can be performed via Etherscan as all contracts are verified on-chain and audited. Please see a list of all the mainnet Bath Tokens [here](https://docs.rubicon.finance/rubicon-docs/contracts/deployments#optimism-mainnet-rubicon-v1).

## Future Works
The storing of Bath Token addresses will be overhauled to be dynamic at the protocol level (and here when fetching as presently a hardcoded string array is used) in future updates. Moreover, in the next PR, we will incorporate custom logos and more information to improve the UX.

Please let me know if I can help the process or provide any additional information - we are pumped to be on Zapper!
